clang/include/clang/Sema/SemaOpenMP.h

```diff
@@ -32,6 +32,7 @@
 #include "clang/Basic/Specifiers.h"
 #include "clang/Sema/DeclSpec.h"
 #include "clang/Sema/Ownership.h"
+#include "clang/Sema/ParsedAttr.h"
```

Comment:
Could this be a forward decl?

Reply:
Yeah, I totally forgot about forward decls while preparing this PR. I fixed that for the things added in this PR, leaving the rest for later.